_run/kube/deployment.yaml

```diff
@@ -19,16 +19,14 @@ profiles:
       memory: 16Mi
       storage: 128Mi
   placement:
-    westcoast:
-      attributes:
-        region: us-west
+    global:
```

Comment:
was this from trying to fix something?

Reply:
Enables the deployment to match against the attributes of the Provider being run in the integration test.